lambdas/select-cri/src/main/java/uk/gov/di/ipv/core/selectcri/SelectCriHandler.java

```diff
@@ -241,47 +241,34 @@ private Optional<JourneyResponse> getCriResponse(
             String journeyId,
             String userId)
             throws ParseException {
-        CredentialIssuerConfig criConfig = configService.getCredentialIssuer(criId);
 
+        CredentialIssuerConfig criConfig = configService.getCredentialIssuer(criId);
         Optional<VcStatusDto> vc = getVc(currentVcStatuses, criConfig.getAudienceForClients());
+
         if (vc.isEmpty()) {
             if (userHasNotVisited(visitedCredentialIssuers, criId)) {
-                CredentialIssuerConfig passportConfig =
-                        configService.getCredentialIssuer(passportCriId);
-                if (criId.equals(dcmawCriId)
-                        && getVc(currentVcStatuses, passportConfig.getAudienceForClients())
-                                .isPresent()) {
-                    LOGGER.info("User already has a passport VC so continuing a web journey");
+                if (criId.equals(dcmawCriId) && hasPassportVc(currentVcStatuses)) {
+                    LOGGER.info("User already has a passport VC, continuing a web journey");
                     return Optional.of(
                             getNextWebJourneyCri(
                                     visitedCredentialIssuers, currentVcStatuses, userId));
                 }
                 return Optional.of(getJourneyResponse(journeyId));
             }
-            var message =
-                    new StringMapMessage()
-                            .with(
-                                    "message",
-                                    "User has a previous failed visit to a cri due to an oauth error")
-                            .with("criId", criId);
-            LOGGER.info(message);
-
-            if (criId.equals(dcmawCriId)) {
-                LOGGER.info("Reverting app user to the web journey");
-                return Optional.of(
-                        getNextWebJourneyCri(visitedCredentialIssuers, currentVcStatuses, userId));
-            }
-
-            return Optional.of(getJourneyPyiNoMatchResponse());
-        } else if (Boolean.FALSE.equals(vc.get().getIsSuccessfulVc())) {
-            var message =
-                    new StringMapMessage()
-                            .with(
-                                    "message",
-                                    "User has a previous failed visit to a cri due to a failed identity check")
-                            .with("criId", criId);
-            LOGGER.info(message);
+            LOGGER.info(
+                    "User has a previous failed visit to a CRI due to an OAuth error for CRI ID {}",
```

Comment:
We explicitly don't what to have values interpolated into log messages. We want to use structured logging,  separating out the values into their own fields.

message.message is not ideal though. Where a better field name does not exist we should use `message.description`